README.md

```diff
@@ -125,4 +125,4 @@ Feel free to check the [issues page](https://github.com/Olivier-Kango/vet_clinic
 This project is [MIT](./LICENSE.txt) licensed.
 
 
```

Comment:
- [ ] `schema.sql file`: As required for the project, kindly make sure you add the schema.sql file code lines of:
  - A new table with the name `owners` with their expected columns and the correct type for each one as in the **[project's requirements.](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md)**
  - Another table with the name `species` with their expected columns and the correct type on each column
  - You add the code you used to modify the `animals` table on how:
    - Remove column `species`
    - Add column `species_id` which is a foreign key referencing the `species` table
    - Add column `owner_id` which is a foreign key referencing the `owners` table